console/program/src/data/plaintext/to_fields_raw.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2019-2025 Provable Inc.
+// This file is part of the snarkVM library.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use super::*;
+
+impl<N: Network> ToFieldsRaw for Plaintext<N> {
+    /// Returns this plaintext as a list of field elements using the raw bits.
+    fn to_fields_raw(&self) -> Result<Vec<Self::Field>> {
+        // Encode the data as little-endian bits without variant or identifier bits.
+        let mut bits_le = self.to_bits_raw_le();
+        // Adds one final bit to the data, to serve as a terminus indicator.
+        // During decryption, this final bit ensures we've reached the end.
+        bits_le.push(true);
```

Comment:
Won't this terminus indicator make it impossible to encode external messages as an Aleo Plaintext? It is also present in the circuit code.

CC @Roee-87 as it may come up if you're already testing on the SDK side.